_posts/2025-04-19-navigating-pagination-with-hybrid-query.md

```diff
@@ -199,14 +199,12 @@ For example:
 
 Custom routing can lead to uneven data distribution across shards, resulting in some shards returning zero results, which reduces the total number of hybridized results.
 
-### Some shards may contain all the relevant results
+Additionally, some shards may contain all the relevant results. If a shard contains more than 20 results per subquery, it can still return only 20 results (based on `pagination_depth`). As a result, the coordinator node will receive fewer results for hybridization, reducing the final search result size.
```

Comment:
In continuation with the earlier example, if some shards may contain all the relevant results due to custom routing. If a shard contains more than 20 results per subquery, it can still return only 20 results (based on pagination_depth). As a result, the coordinator node will receive fewer results for hybridization, reducing the final search result size.